linkerd2-proxy/src/rt.rs

```diff
@@ -64,3 +64,37 @@ pub(crate) fn build() -> Runtime {
         .build()
         .expect("failed to build basic runtime!")
 }
+
+// Spawns a task to scrape metrics for the given runtime at a regular interval.
```

Comment:
```suggestion
/// Spawns a task to scrape metrics for the given runtime at a regular interval.
```

Reply:
great catch, thank you!

Reply:
...and rebased that commit with a sign-off. :heavy_check_mark: